scripts/check_unit.sh

```diff
@@ -53,11 +53,16 @@ check_exit_code $DOCKER_RM_EXIT_CODE "docker rm AriesMongoDBStorageTest"
 check_exit_code $DOCKER_RM_EXIT_CODE "docker rm AriesEDVStorageTest"
 }
 
-# Running aries-framework-go unit test
+# Running aries-framework-go unit test without ursa
 PKGS=$(go list github.com/hyperledger/aries-framework-go/pkg/... 2> /dev/null | grep -v /mocks | grep -v /aries-js-worker)
 $GO_TEST_CMD $PKGS -count=1 -race -coverprofile=profile.out -covermode=atomic -timeout=10m
 amend_coverage_file
 
+# Running aries-framework-go unit test with ursa
+#PKGS=$(go list -tags ursa github.com/hyperledger/aries-framework-go/pkg/... 2> /dev/null | grep -v /mocks | grep -v /aries-js-worker)
+#$GO_TEST_CMD -tags ursa $PKGS -count=1 -race -coverprofile=profile.out -covermode=atomic -timeout=10m
+#amend_coverage_file
```

Comment:
```suggestion
PKGS=$(go list -tags ursa github.com/hyperledger/aries-framework-go/pkg/... 2> /dev/null | grep -v /mocks | grep -v /aries-js-worker)
$GO_TEST_CMD -tags ursa $PKGS -count=1 -race -coverprofile=profile.out -covermode=atomic -timeout=10m
amend_coverage_file
```

I don't mind adding ursa unit tests in the build.

Reply:
Added a separate `make unit-test-ursa` target and a separate job in the CI pipeline to run unit tests with the `ursa` build tag.
There is a reason why it's not easily possible to get rid of `ursa` build tag and just run ursa/CL test as other unit tests: ursa-wrapper-go requires `libursa` to be installed. Current development process and CI unit tests are not based on Docker, so that it may be challenging to require libursa library to be installed for every developer. 
CI job for the `ursa` tag is run in a container (`ghcr.io/hyperledger/ursa-wrapper-go/uwg-build` for now) where `libursa` is already installed.

Reply:
thanks for the explanation